package.json

```diff
@@ -72,8 +73,9 @@
     "dev": "rollup -c rollup.config.js --watch",
     "update-icons": "node ./src/scripts/update-icons.js",
     "check": "svelte-check --compiler-warnings \"missing-custom-element-compile-options:ignore\"",
-    "transform-tokens": "node ./src/tokens/transformTokens.js && tsc -p ./tsconfig-tokens.json",
+    "transform-tokens": "node ./src/tokens/transformTokens.js && tsc -p ./tsconfig-tokens.json && npm run transform:icons",
     "test": "jest tests --coverage=false",
+    "transform:icons": "node ./src/scripts/gen-skia-icons.js",
     "transform:android": "node ./examples/android/build.js",
     "transform:ios": "node ./examples/ios/build.js",
```

Comment:
Unrelated to this PR, can we remove these?

Reply:
https://github.com/brave/leo/pull/224